src/components/DateRangePicker/Selector.js

```diff
@@ -0,0 +1,116 @@
+const React = require('react');
+const moment = require('moment');
+const PropTypes = require('prop-types');
+
+const Icon = require('../Icon');
+
+class Selector extends React.Component {
+  static propTypes = {
+    currentDate: PropTypes.string,
+    handleNextClick: PropTypes.func,
+    handlePreviousClick: PropTypes.func,
+    setCurrentDate: PropTypes.func
+  };
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <div style={styles.container}>
+        <Icon
+          elementProps={{ onClick: this.props.handlePreviousClick }}
+          size={20}
+          style={styles.calendarHeaderNav}
+          type='caret-left'
+        />
+        <div>
+          {this.props.currentDate}
+        </div>
+        <Icon
+          elementProps={{ onClick: this.props.handleNextClick }}
+          size={20}
+          style={styles.calendarHeaderNav}
+          type='caret-right'
+        />
+      </div>
+    );
+  }
+
+  styles = () => {
+    return {
+      container: {
+        display: 'flex'
+      },
+      calendarHeaderNav: {
+        width: 35,
+        cursor: 'pointer'
+      }
+    };
+  }
+}
+
+class MonthSelector extends React.Component {
```

Comment:
Is there a reason MonthSelector & YearSelector can't be configured into a single component?